docs/installation/presto-admin-configuration.rst

```diff
@@ -43,7 +43,7 @@ be used:
 
 Note that ``java8_home`` is not set by default.  It only needs to be set if
 Java 8 is in a non-standard location on the Presto nodes.  The property is used
-to tell the presto rpm where to find Java 8.
+to tell the Presto RPM where to find Java 8.
 
 You can also specify some but not all of the properties. For example, the
 default configuration is for a single-node installation of Presto on the same
```

Comment:
it's a continuation of the line before :) I was just updating the capitalization here